### ISSUE TITLE
partially grouped neume components

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/Neon#642]
Selecting a subset of the neume components of an already grouped neume and "grouping neume components" causes the order of colours in the highlighting to change, as if an invisible intervening neume has been created. Does this matter?

**Comment by JRegimbal** [comment]
I haven't been able to get an example with both a before and after MEI file of this change in color, but since neume grouping isn't destructive it *shouldn't* cause a problem. Of course this should be investigated more to actually see what (if anything) changes in the MEI file or if this is a quirk of the highlighting code.

**Comment by JoyfulGen** [comment]
Just for fun, I tested this in Einsie 019v. I grouped the neume components B and C (puncta) of the light blue neume below:
<img width="620" alt="019v Secret yellow neume" src="https://user-images.githubusercontent.com/83373378/152803753-738b1fd7-8bd9-4cd9-ab1f-37432ab6143a.png">
As a consequence, the colours of the neumes following the light blue boy changed. There's now a yellow neume missing between the green and dark blue neumes.

Here's a side-by-side of the MEI files of the folio before and after I grouped the neumes (confusingly, the after file is on the left--sorry about that).
<img width="1350" alt="19v side by side" src="https://user-images.githubusercontent.com/83373378/152804149-390a1126-edc7-491a-a0c8-4b0f96847f24.png">
And here are the complete MEI files:
[Einsie_019v_before.zip](https://github.com/DDMAL/Neon/files/8015913/Einsie_019v_before.zip)
[Einsie_019v_after.zip](https://github.com/DDMAL/Neon/files/8015914/Einsie_019v_after.zip)

Weirdly, the light blue neume is identical. The only difference is that extra neume after the green virga. It doesn't have any neume components, which explains why it doesn't appear on Neon.

I don't think this is a huge problem, because it doesn't seem to affect any other neume, and the syllables are fine. It is pretty quirky though!
